solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -227,6 +252,7 @@ def process_forecast_observations(forecast_observations, filters, data,
                 interval_label=fxobs.forecast.interval_label,
                 valid_point_count=len(forecast_values),
                 validation_results=val_results,
+                preprocessing_results=preproc_results,
```

Comment:
Can you add checks of this in a process_forecast_observations test?

Reply:
I added checks that all objects of `validation_results` are of type `ValidationResult` and `preprocessing_results` are of type `PreprocessingResult`... something else you have in mind?